server/webhook.go

```diff
@@ -192,39 +226,48 @@ func parse(in io.Reader, linkf func(w *JIRAWebhook) string) (*parsedJIRAWebhook,
 	issue := parsed.mdIssueType() + " " + linkf(parsed.JIRAWebhook)
 	switch parsed.WebhookEvent {
 	case "jira:issue_created":
+		parsed.events = parsed.events | eventIssueCreated
```

Comment:
I feel like the concern of populating this bitset should be removed from everything else that's going on here. Then the code here can use this bitset.